web/css/components/popup.css

```diff
@@ -0,0 +1,43 @@
+.container__popup {
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  width: 784px;
+  padding: 40px;
+  gap: 8px;
+  border-radius: 24px;
+  background-color: #f2f6f9;
+}
+
+.container_img {
```

Comment:
where is the figure settings settings the image size ?
check what's already set for figure>img in others components